skimage/feature/corner.py

```diff
@@ -123,25 +123,29 @@ def hessian_matrix(image, sigma=1, mode='constant', cval=0):
     mode : {'constant', 'reflect', 'wrap', 'nearest', 'mirror'}, optional
         How to handle values outside the image borders.
     cval : float, optional
+        else:optional
         Used in conjunction with mode 'constant', the value outside
         the image boundaries.
+    order : {'xy', 'rc'}, optional
+        this parameter allows for the use of reverse or forward order of
+        returned matrix values
 
     Returns
     -------
-    Hxx : ndarray
+    Hrr : ndarray
         Element of the Hessian matrix for each pixel in the input image.
-    Hxy : ndarray
+    Hrc : ndarray
         Element of the Hessian matrix for each pixel in the input image.
-    Hyy : ndarray
+    Hcc : ndarray
         Element of the Hessian matrix for each pixel in the input image.
 
     Examples
     --------
     >>> from skimage.feature import hessian_matrix
     >>> square = np.zeros((5, 5))
     >>> square[2, 2] = 4
-    >>> Hxx, Hxy, Hyy = hessian_matrix(square, sigma=0.1)
-    >>> Hxy
+    >>> Hrr, Hrc, Hcc = hessian_matrix(square, sigma=0.1)
```

Comment:
This docstring example is incorrect, because without order='rc', it will return Hxx, Hxy, Hyy. So I would include order='rc' in the docstring call.